### PR TITLE
Implement Contributions Page MVP

### PR DESCRIPTION
**Pull request by topocount**
## Motivation and Context

Our Users want to communicate their work and allow it to appear inline during the give circle

## Description

This is a minimally viable feature set. The following remains to be implemented:

- link syntax highlighting (this should probably be bundled with markdown support)
- opengraph link previews

Features WIP in this PR:
- [x] nav buttons between contributions
- [x] autosaving (which could be implemented in this PR if folks insist)

## Test and Deployment Plan

This is currently implemented as a hidden page, accessible in the `/circles/{circle_id}/contributions` route only.

unit tests can be added but this was a significant implementation and Im keen to get some feedback on the current usability.

to test this navigate to various circles and append the `/contributions` route. Open epochs and no existing epochs should allow for the modification of contributions. This should not be allowable on closed epochs (though the ui still indicates the possiblity of updating the contribution, which i need to fix.)

Let me know if any errors arise. I've tested many different contexts and patterns but might have missed something

## Screenshots (if appropriate):

Peek 2022-09-28 12-19

## Reviewers

@levity 
@wingfeatherwave 
@CryptoGraffe

### CODE REVIEW

**Comment by crabsinger** [comment]
Just pushed this to `staging` to get hasura staging to support this for testing on vercel

**Comment by wingfeatherwave** [comment]
@topocount in staging, the new contributions that I add aren't appearing in the list of future contributions

https://user-images.githubusercontent.com/100873710/193122789-93761665-f747-4865-b32b-6a337f86bd40.mov

**Comment by topocount** [comment]
> @topocount in staging, the new contributions that I add aren't appearing in the list of future contributions

@wingfeatherwave just seeing this! Thanks will investigate

**Comment by topocount** [comment]
@wingfeatherwave the latest updates (where this is fixed) weren't successfully deployed due to some issues outside of this branch. Should be good now: https://vercel.com/coordinape/coordinape/deployments?teamSlug=coordinape&project=coordinape&filterBranch=staging

File: hasura/metadata/actions.yaml [hunk 1]
```diff
@@ -121,15 +121,6 @@ actions:
...
       - role: user
-  - name: deleteContribution
-    definition:
-      kind: synchronous
-      handler: '{{HASURA_API_BASE_URL}}/actions/actionManager'
-      headers:
-      - name: verification_key
-        value_from_env: HASURA_EVENT_SECRET
-    permissions:
-    - role: user
```
Comment (topocount) [inline hasura/metadata/actions.yaml:132]: this is a duplicate entry, probably due to some wonky merging

**Review by wingfeatherwave — DISMISSED** [review]

**Review by wingfeatherwave — APPROVED** [review]
yay!  Nice to have the next prev buttons

**Pull request merged**